src/hooks/popoverConstants.ts

```diff
@@ -0,0 +1,8 @@
+export enum PLACEMENTS {
```

Comment:
It's convention to call enums with pascal-case, and in singulars (same for the values, but not critical):

```suggestion
export enum Placement {
```